data/json/items/gunmod/accessories.json

```diff
@@ -132,5 +132,33 @@
     "location": "belt clip",
     "mod_targets": [ "rugerlcp", "kp32", "kp3at", "kpf9", "cop_38", "moss_brownie" ],
     "flags": [ "IS_ARMOR", "SKINTIGHT", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "12_gauge_shell_holder",
+    "type": "GUNMOD",
+    "name": { "str": "12 gauge shell holder" },
+    "description": "Designed to be affixed to the side of a shotgun's receiver, this polymer shotshell holder can store six 12 gauge shells for easy reloading.",
+    "weight": "16 g",
+    "volume": "60 ml",
+    "price": 2000,
+    "price_postapoc": 50,
+    "install_time": "30 s",
+    "material": [ "plastic" ],
+    "flags": [ "NOT_MAGAZINE", "TARDIS" ],
```

Comment:
Why does this need TARDIS flag, a magazine well that covers the volume of the shotshell that isn't poking out should suffice 🤔

Reply:
Yeah, either the pocket is rigid and takes up the full space even without contents or it's not rigid and every loaded shell increases the size accordingly. From the images I'd expect the latter.

Reply:
It doesn't. I had it on there while trying to figure out a way to keep the shells from falling out when a full holder was removed, but it didn't work and I haven't gotten around to fixing it. Note that this PR is marked draft and is not ready for review.